build.tcl

```diff
@@ -378,31 +391,37 @@ proc _install {args} {
 		set version {}
 	    }
 
+	    set namevers $dstl/[pkgdirname $name $version]
```

Comment:
Seeing a number changes to use `file join`, why not here ?

Reply:
I've been changing things not to use [file join], rather than to use it.